modules/carto/src/api/layer-map.ts

```diff
@@ -202,31 +209,29 @@ export function getLayer(
 }
 
 export function layerFromTileDataset(
-  scheme: string,
-  type?: MapType
+  type: string
```

Comment:
@jmgaya  I think it would good to add the type, especially as we already have a `MapType` from the API which could be confusing. 

Also rename function to `layerFromMapType`

Reply:
On it 👍